### PR TITLE
Upgrade agent to latest

### PR DESCRIPTION
**Pull request by fwang-teradici**

### CODE REVIEW

File: end-user-application-machines/azuredeploy.json [hunk 1]
```diff
@@ -103,6 +103,12 @@
           "metadata": {
             "description": "PCoIP Standard Agent License Registration Code."
           }
+        },
+        "AgentChannel": {
+          "type": "string",
+          "metadata": {
+            "description": "from the channel to download PCoIP Agent Installer."
```
Comment (bill-dall) [inline end-user-application-machines/azuredeploy.json]: Should use the description from the other template I think.

File: end-user-application-machines/new-agent-vm/Install-PCoIPAgent.ps1 [hunk 2]
```diff
@@ -228,8 +228,20 @@ Configuration InstallPCoIPAgent
...
+                $installerFileName = "PCoIP_agent_release_installer_standard.exe"
+
+				if (! $using:isSA) {
+					$installerFileName = "PCoIP_agent_release_installer_graphics.exe"
+				}
+
                 $pcoipAgentInstallerUrl = $using:pcoipAgentInstallerUrl
-                $installerFileName = [System.IO.Path]::GetFileName($pcoipAgentInstallerUrl)
+
+				if (! $pcoipAgentInstallerUrl.EndsWith('\') ) {
```
Comment (bill-dall) [inline end-user-application-machines/new-agent-vm/Install-PCoIPAgent.ps1]: Backslash for a URL?

Comment (fwang-teradici) [reply end-user-application-machines/new-agent-vm/Install-PCoIPAgent.ps1]: ops,  I will update it

File: azuredeploy.json [hunk 3]
```diff
@@ -85,6 +85,14 @@
           "description": "Id of the keyvault where user credentials for the deployment are stored."
       }
     },
+    "AgentChannel": {
+      "type": "string",
+      "defaultValue": "stable",
+      "allowedValues": ["stable","beta","dev"],
+      "metadata": {
+        "description": "PCoIP Agent channel to use for installer download ."
```
Comment (bill-dall) [inline azuredeploy.json]: small nit - I see there's a space between download and .

Comment (fwang-teradici) [reply azuredeploy.json]: yes, it is. I will remove the space

File: end-user-application-machines/azuredeploy.json (hunk 1 above)
Comment (bill-dall) [reply end-user-application-machines/azuredeploy.json]: @fwang-teradici  - sorry I'm being annoying but I think you missed this comment?

**Pull request merged**